engine/planner.go

```diff
@@ -9,11 +9,10 @@ import (
 	"sort"
 	"time"
 
+	"github.com/dchest/uniuri"
```

Comment:
the convention is to group internal (e.g. drone dependencies) and external packages separately, which is why uniuri is grouped with zerolog.

Reply:
I am going to revert the `go fmt` and `drone fmt` changes so it doesn't add weird changes that aren't really part of this PR.